### ISSUE TITLE
codified text not themed

### ISSUE THREAD

**Issue by ashkitten** [glowing-bear/glowing-bear#1056]
i use the dark theme and the recent addition of a `<code>` wrapper for text surrounded by backticks makes it very hard to read.

**Comment by lorenzhs** [comment]
Hm, it's near-white on dark gray, with an [AAA rating](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) of 8.83 so it shouldn't be particularly hard to read? Maybe try a force refresh, could a cache issue with the CSS file?

Other than that, you're right in that it's currently not adapted to other themes

**Comment by ashkitten** [comment]
it's difficult to read because it's so bright against the background, i mean. it's fine in a light theme.

**Comment by lorenzhs** [comment]
sounds like a cache issue, because it's supposed to look like this: ![Screenshot from 2019-08-02 19-04-10](https://user-images.githubusercontent.com/142205/62386486-55a2e880-b558-11e9-8ed8-9996c9470e0e.png)

**Comment by ashkitten** [comment]
this is what i'm seeing
![image](https://user-images.githubusercontent.com/9281956/62388563-7c3e3080-b512-11e9-997b-865cf3d9514c.png)

**Comment by lorenzhs** [comment]
Ah shit. I just checked my custom CSS and it contains
```css
code {
  background-color: #444;
  color: #fff;
}
```
:man_facepalming:

**Comment by lorenzhs** [comment]
should probably just add that to the dark theme // cc @spapas

**Comment by spapas** [comment]
Hello the only css I added for codify filter was something like 
```
code {
    padding: 0px 2px;
} 
```

to make it behave better for inline code. 

So all `code` theming is inherited from bootstrap's defaults. I didn't want to do any theming because I'm afraid people won't like my color decisions (I actually use the dark theme with the default css and it doesn't bother me that much).

However I think that the css snippet @lorenzhs proposed should be appended to all dark-backgrounded glowing-bear themes, not only `dark.css`.

**Comment by lorenzhs** [comment]
Fixed in ac6dca959d7421682629f5550cccad69c9484b03
